src/services/neo4jHelpers.js

```diff
@@ -1,13 +1,18 @@
+'use strict';
 const R = require('ramda');
 const Boom = require('boom');
 const driver = require('../db');
 
-function handleSuccess (session, results) {
+function handleSuccess (session, results, returnKey/*: ?string*/) {
```

Comment:
Fyi i changed this in Master so this change isnt needed. Check out this file on Master to see what i did.